testing/tests/DevExpress.knockout/textBox.tests.js

```diff
@@ -0,0 +1,29 @@
+const $ = require('jquery');
```

Comment:
let's remove $ from this test?

Reply:
Was updated here: [83e8e4e131d792882cdedfb3a6dadb0eb4b88383].